project-base/storefront/components/Layout/Header/AutocompleteSearch/AutocompleteSearch.tsx

```diff
@@ -100,6 +112,8 @@ export const AutocompleteSearch: FC = () => {
                             areAutocompleteSearchDataFetching={areAutocompleteSearchDataFetching}
                             autocompleteSearchQueryValue={searchQueryValue}
                             autocompleteSearchResults={searchData}
+                            favoritesData={favoritesData}
+                            showFavorites={showFavorites}
```

Comment:
I don't know where is the cause, but when you click into the search input, box with the favorites is shown. 
I would
- remove the numbers (they have no information value imho)
- reword the titles - `Favorite` everywhere cause a visual smog and it may imply something else (why my favorite/wishlisted/frequently bought products are not shown?) 
- input itself is grayed out - it looks like I cannot search anymore

Reply:
> I don't know where is the cause, but when you click into the search input, box with the favorites is shown.

Well that was the point of the user-story, and one of the acceptance criteria :smile: Or is there any misunderstanding?

Reply:
c5689ea6e3 - number removed, "favorite" removed, input is not greyed-out anymore :slightly_smiling_face:

Reply:
> > I don't know where is the cause, but when you click into the search input, box with the favorites is shown.
> 
> Well that was the point of the user-story, and one of the acceptance criteria 😄 Or is there any misunderstanding?

No, that's just my potato English.  I wanted to pinpoint the three things you've already fixed 😄